apps/aehttp/src/aehttp_app.erl

```diff
@@ -12,6 +12,8 @@
 -define(DEFAULT_SWAGGER_EXTERNAL_LISTEN_ADDRESS, <<"0.0.0.0">>).
 -define(DEFAULT_SWAGGER_INTERNAL_PORT, 8143).
 -define(DEFAULT_SWAGGER_INTERNAL_LISTEN_ADDRESS, <<"127.0.0.1">>).
+-define(DEFAULT_ROSETTA_PORT, 8243).
```

Comment:
This is an entirely different default from the one in the aeternity config schema, and the sys.config files.
In general, at least I tend to rely on the schema defaults alone, esp. given that we can now override those values using OS env variables.

Reply:
Just following the existing convention. Re-doing the whole config thing is definitely needed, but I prefer to not do it here